<launch>
  <!-- SIMULATION -->
  <include file="$(find turbot)/launch/simulation.launch"/>


  <!-- SET MISSION -->
  <!-- Load trajectory in the parameter server -->
  <rosparam command="load" file="$(find usbl_position)/missions/line_short.yaml"/>
  <!-- Call service -->
  <node pkg="usbl_position" name="traj2bag" type="traj2bag" output="screen"/>



</launch>
